docs/tutorials/new_edv.rst

```diff
@@ -140,7 +140,9 @@ where
 - ``default``: (optional) default value to use in initialization.
 
 For further details, see the documentation of
-:class:`~baseband.vlbi_base.header.HeaderParser`.
+:class:`~baseband.vlbi_base.header.HeaderParser`.  If the EDV specifies a sync
+pattern, it must be included as the ``_sync_pattern`` private attribute (or
```

Comment:
Don't like this much: how about we let the meta class set the `_sync_pattern` if it is present in the header parser? I also wonder about perhaps having too many indirections in `HeaderProperty` and `HeaderPropertyGetter`.